pkg/es/config/config.go

```diff
@@ -282,21 +300,21 @@ func (c *Configuration) ApplyDefaults(source *Configuration) {
 
 // GetIndexRolloverFrequencySpansDuration returns jaeger-span index rollover frequency duration
 func (c *Configuration) GetIndexRolloverFrequencySpansDuration() time.Duration {
-	return getIndexRolloverFrequencyDuration(c.IndexRolloverFrequencySpans)
+	return GetIndexRolloverFrequencyDuration(c.Indices.Spans.RolloverFrequency)
 }
 
 // GetIndexRolloverFrequencyServicesDuration returns jaeger-service index rollover frequency duration
 func (c *Configuration) GetIndexRolloverFrequencyServicesDuration() time.Duration {
-	return getIndexRolloverFrequencyDuration(c.IndexRolloverFrequencyServices)
+	return GetIndexRolloverFrequencyDuration(c.Indices.Services.RolloverFrequency)
 }
 
 // GetIndexRolloverFrequencySamplingDuration returns jaeger-sampling index rollover frequency duration
 func (c *Configuration) GetIndexRolloverFrequencySamplingDuration() time.Duration {
-	return getIndexRolloverFrequencyDuration(c.IndexRolloverFrequencySampling)
+	return GetIndexRolloverFrequencyDuration(c.Indices.Sampling.RolloverFrequency)
 }
 
 // GetIndexRolloverFrequencyDuration returns the index rollover frequency duration for the given frequency string
-func getIndexRolloverFrequencyDuration(frequency string) time.Duration {
+func GetIndexRolloverFrequencyDuration(frequency string) time.Duration {
```

Comment:
this change seems unnecessary, this was a private helper function not used anywhere else

Reply:
I left as public here because we need to convert string to duration in the `reder.go`